telegram_packs/root.txt

```diff
@@ -0,0 +1,83 @@
+U1f31e-frogSun
```

Comment:
whats up with this file?

Reply:
the unicode prefix determines the emoji, so no further mapping is needed.
I could maybe just pull the project root and drop this file though... 
Will have a look into that.

Reply:
fixed on the bot side